<launch>

  <arg name="directory" default="" />
  <arg name="topics" default="odom amcl_pose /diagnostics" />

  <node name="record" pkg="bwi_logging" type="record"
        args="$(arg topics)">
    <param name="directory" value="$(arg directory)"/>
  </node>

</launch>
